src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -1,6 +1,62 @@
 package frc.robot.subsystems.shooter;
 
+import frc.robot.Constants;
+import lib.webconstants.LoggedTunableNumber;
+
 public class ShooterConstants {
-    public static final int FRONT_GEAR_RATIO = 0;
-    public static double SETPOINT_TOLERANCE = 0.05; // Percentages
+    public static final double GEAR_RATIO = 1.0;
+    public static final double SETPOINT_TOLERANCE = 0.05; // Percentages
+    public static final double MOMENT_OF_INERTIA = 0.08;
+
+    public static final LoggedTunableNumber TOP_P = new LoggedTunableNumber("Shooter/Top P");
+    public static final LoggedTunableNumber TOP_I = new LoggedTunableNumber("Shooter/Top I");
+    public static final LoggedTunableNumber TOP_D = new LoggedTunableNumber("Shooter/Top D");
```

Comment:
3338e24

Reply:
There are still a few places where it's `K{lowercase}` instead of `k{uppercase}`, all of them should be fixed. Note that the constant name itself doesn't have to follow this convention (there it's fine to leave all caps), but the string should be e.g. `kV`.

Reply:
d0fa7d9

Reply:
Great!